src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -299,6 +299,20 @@ void assertAttributeValue(String attributeName, Collection<String> permittedValu
     }
   }
 
+  /**
+   * Checks if a required attribute is not null or empty
+   *
+   * @param attributeName name of attribute that will be checked.
+   * @throws InvalidInputException
+   */
+  void assertRequired(String attributeName) throws InvalidInputException {
```

Comment:
Not a good name for this method, I don't know what I'm asserting if I just see the method call.